amun/inspect.py

```diff
@@ -88,6 +88,22 @@ def _gather_os_release():
     return result
 
 
+def _gather_runtime_environment(os_release: dict):
+    """Gather information about runtime environment."""
+    return  {
+        "cuda_version": None,
+        "hardware": {
+            "cpu_family": None,
+            "cpu_model": None
+            },
```

Comment:
I'm didn't find out how to take this information from specification @fridex :/

Reply:
Hm... in that case it could be better to move this hardware discovery to amun-hwinfo - https://github.com/thoth-station/amun-hwinfo/blob/master/app.py and utilize https://pypi.org/project/py-cpuinfo/ On syncing side we will need to combine entries from here and amun-hwinfo, but it sounds like a good approach.

Reply:
on the syncing logic side, actually everything would be here, because in `hwinfo` we will output information from `https://pypi.org/project/py-cpuinfo/ `, which will go in `hwinfo` here. In the syncing side only the `python_version` should be added, unless we add it here as well.

Reply:
> > Hm... in that case it could be better to move this hardware discovery to amun-hwinfo - https://github.com/thoth-station/amun-hwinfo/blob/master/app.py and utilize https://pypi.org/project/py-cpuinfo/ On syncing side we will need to combine entries from here and amun-hwinfo, but it sounds like a good approach.
> 
> As results of amun-hwinfo are parsed in inspect.py, the results can be used to construct runtime environment entries in inspect.py directly, not in thoth-storage.
> 
> https://github.com/thoth-station/amun-api/blob/930d50cfe26b2f43645f3d59cccb887845bd3c29/amun/inspect.py#L96

Yes agree, on the syncing logic side, almost everything would be here, because in `hwinfo` we will output information from `https://pypi.org/project/py-cpuinfo/ `, which will go in `hwinfo` here. The missing thing is `python_version` that should be added in syncing side, unless we add it here as well.

Reply:
OK, so for simplicity - let's construct the whole `runtime_environment` here so the syncing logic can directly picks it. Ack?